community/installation-guides/panel/debian10.md

```diff
@@ -25,6 +25,10 @@ systemctl enable mariadb
 
 ### PHP 7.4
 ```bash
+# Add additional repositories for PHP, Redis, and MariaDB
+sudo wget -O /etc/apt/trusted.gpg.d/php.gpg https://packages.sury.org/php/apt.gpg
```

Comment:
Sudo is not used in any of the other command examples either. Please remove it. Preferably we'd use curl since it was installed previously